src/main/resources/hudson/plugins/warnings/parser/Messages.properties

```diff
@@ -266,3 +266,11 @@ Warnings.RTTests.TrendName=RT Tests Trend
 Warnings.ClangTidy.ParserName=Clang-Tidy
 Warnings.ClangTidy.LinkName=Clang-Tidy
 Warnings.ClangTidy.TrendName=Clang-Tidy Warnings Trend
+
+Warnings.SonarQubeIssues.ParserName=SonarQube Issues
+Warnings.SonarQubeIssues.LinkName=SonarQube Issues
+Warnings.SonarQubeIssues.Trend=SonarQube Issues Warnings Trend
+
+Warnings.SonarQubeDiff.ParserName=SonarQube Diff
+Warnings.SonarQubeDiff.LinkName=SonarQube Diff
+Warnings.SonarQubeDiff.Trend=SonarQube Diff Warnings Trend
```

Comment:
You can have icons and a default file name pattern in the next major release, does it make sense to define one for SonarQube?

See [example](https://github.com/jenkinsci/warnings-plugin/blob/3.0/src/main/java/io/jenkins/plugins/analysis/warnings/SpotBugs.java).

Reply:
That'd be nice, maybe put [these icons](https://github.com/SonarSource/sonar-scanner-jenkins/tree/master/src/main/webapp/images) from the SQ scanner plugin?
And a default pattern of "**/target/sonar/sonar-report.json".

Should I change branch and extend StaticAnalysisTool for that though? :'D

Reply:
Frankly, I don't have much time on my hands, but maybe I can do it. What's there to be done? :0

Reply:
Basically, the same PR for the https://github.com/jenkinsci/analysis-model project. You need to extend from [AbstractParser|https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/AbstractParser.java] and need to create a [Report](https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/parser/jcreport/JcReportParser.java#L30). The test needs to extend from [AbstractParserTest](https://github.com/jenkinsci/analysis-model/blob/master/src/test/java/edu/hm/hafner/analysis/parser/JcReportParserTest.java#L89)

Reply:
Ok, I'll make the changes to the analysis-model project and PR. Is there anything else to do in this PR? :0 Thank you for your attention!